pkg/machine/apple/machine.go

```diff
@@ -0,0 +1,70 @@
+//go:build arm64 && !windows && !linux
+// +build arm64,!windows,!linux
```

Comment:
This must use the darwin build tag, not linux and windows would also include BSDs for example